src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -415,12 +415,12 @@ private MethodIL Get_CORINFO_METHOD_INFO(MethodDesc method, MethodIL methodIL, C
             return methodIL;
         }
 
-        private void Get_CORINFO_SIG_INFO(MethodDesc method, CORINFO_SIG_INFO* sig, bool isFatFunctionPointer = false)
+        private void Get_CORINFO_SIG_INFO(MethodDesc method, CORINFO_SIG_INFO* sig, bool isFatFunctionPointerOrInstantiatingStub = false)
```

Comment:
We could name it `suppressHiddenArgument` or something like that to keep it short.

Reply:
Good idea, will do.

Reply:
Renamed in 2nd commit. Looking at the results of my overnight local Pri#1 run I see that the change fixes about a thousand tests, reducing the number of remaining failures from 2298 to 1275 (out of the total number of 10511 tests) and bumping up the overall pass rate for this test suite from 78% to 88%.